cmd/mimir/help-all.txt.tmpl

```diff
@@ -669,12 +669,10 @@ Usage of ./cmd/mimir/mimir:
     	Max size - in bytes - of a gap for which the partitioner aggregates together two bucket GET object requests. (default 524288)
   -blocks-storage.bucket-store.posting-offsets-in-mem-sampling int
     	Controls what is the ratio of postings offsets that the store will hold in memory. (default 32)
+  -blocks-storage.bucket-store.series-fetch-preference float
+    	Increasing the series preference results in fetching more series to fulfil a Series request and reduces the volume of postings fetches. Reducing the preference results in the opposite. Increase to reduce the rate of fetched series bytes (see "Mimir / Queries" dashboard) or API calls to the object store. Must be a positive floating point number. (default 0.75)
```

Comment:
```suggestion
    	Increasing the series preference results in fetching more series to fulfill a series request and reducing the volume of postings fetches. Reducing the series preference results in the opposite. Increase this setting to reduce the rate of fetched series bytes (see "Mimir / Queries" dashboard) or API calls to the object store. Must be a positive floating point number. (default 0.75)
```

Reply:
@dimitarvdimitrov I think it might help to start off by describing what this setting is before explaining how to modify it. Something like "Controls the number of series to fetch to fulfill a request."

Reply:
yeah that makes sense. Can you take another look? I think other places refer to CLI flags as parameters, so I replaced "setting" with "parameter".